### ISSUE TITLE
using ssd pretrain model to train own model，but result is disappointment.

### ISSUE THREAD

**Issue by 995270418L** [tensorflow/models#7670]
my target:
    my pretrain model is  [ssd_resnet50_fpn_coco](http://download.tensorflow.org/models/object_detection/ssd_resnet50_v1_fpn_shared_box_predictor_640x640_coco14_sync_2018_07_03.tar.gz) my target is two type of bottle, one shape and size is same as the other, only different is it's color. 

description: 
        when my dataset size from 500 images go up to 900 images ( type ratio 1:1),  the result is not satificate me.  in sometimes.  the model will detection one things which color is most same as target but size not equals. and this problem is not decrease with my dataset grow up. and the accuracy is decrease with dataset grow up,  but precision increased. 

my train environment and result:
     GPU: tesla v100
     CPU: intel i7
     steps: 20000
      loss: 0.20
      mAP: 0.74

can anyone help me increase my model's precison ? thanks.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
